lms/services/hubspot/service.py

```diff
@@ -0,0 +1,90 @@
+from datetime import datetime
+
+from hubspot import HubSpot
+from sqlalchemy import select
+from sqlalchemy.exc import MultipleResultsFound
+
+from lms.models.hubspot import HubSpotCompany
+from lms.services.hubspot._client import HubSpotClient
+from lms.services.upsert import bulk_upsert
+
+
+class HubSpotService:
+    def __init__(self, db, client: HubSpotClient, region_code: str):
```

Comment:
Should we add a type annotation to `db`? It seems a complex object, so having some IDE intellisense would help.